src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/quick-start-nrql-condition.mdx

```diff
@@ -0,0 +1,73 @@
+
+![A screenshot showing where you can find the NRQL query builder in New Relic One.](./images/find-builder.png "A screenshot showing where you can find the NRQL query builder in New Relic One.")
+<figcaption>
+  The graph icon at the top includes the **Create alerts** button that will be used in this example. Click the **Open in Query builder** button after using the bottom NRQL query choice to reach the same page.
+</figcaption>
+
+In our example, our condition shows us each host’s CPU utilization. Write out the NRQL query as seen in the image below to track each host individually.
+
+![A screenshot of a NRQL query.](./images/nrql-query.png "A screenshot of a NRQL query.")
+<figcaption>
+  This query finds the average CPU utilization from a system and then breaks it down (facets) by individual host names.
+</figcaption>
+
+Our NRQL query language is a powerful tool that lets you write queries on any entity and its attributes. For more on NRQL, see [NRQL syntax](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/#syntax).
+
+Once you create a valid NRQL query, click on the “Create alert” button.
+
+![A screenshot of a NRQL query with an arrow pointing at the "Create alert" button.](./images/create-alert-button.png "A screenshot of a NRQL query with an arrow pointing at the "Create alert" button.")
+<figcaption>
+  Press enter to enable the "Create alert" button.
+</figcaption>
+
+## Step 2: See your query results [#step2-nrql]
+
+Visualize your NRQL alerts condition on New Relic One to make it easier for you to see how your systems are performing. In our example, our query shows how each host’s average CPU utilization is performing. A violation will open if any host’s average CPU utilization is over 70% and can alert us.
+
+![A screenshot of the New Relic One platform visualizing NRQL query.](./images/show-nrql-graph.png "A screenshot of the New Relic One platform visualizing NRQL query.")
+<figcaption>
+  The above screenshot shows the critical and warning threshold lines. These will appear once you define your thresholds.
+</figcaption>
+
+## Step 3: Create an alerts condition with thresholds [#step3-nrql]
+
+Finish setting up your first alerts condition. Set thresholds to determine when you’re alerted. In this example, our condition checks for any hosts with CPU utilization over 70% for at least 5 minutes. A warning threshold is also added to show us when any host's CPU utilization goes over 67% for at least 5 minutes.
+
+![A screenshot of the New Relic One platform setting up a critical threshold.](./images/nrql-set-threshold.png "A screenshot of the New Relic One platform setting up a critical threshold.")
+
+<figcaption>
+  This is the **Set your condition thresholds** section on the form for creating an alert. Complete the other sections as needed.
```

Comment:
Drop the second sentence. I think you could get more descriptive in this caption.